arch/risc-v/include/irq.h

```diff
@@ -724,24 +716,6 @@ int up_this_cpu(void);
  * Inline Functions
  ****************************************************************************/
 
-static inline_function uintreg_t *up_current_regs(void)
```

Comment:
What is the motivation behind removing this logic, I don't see any functional change between the old and new ?

Reply:
Currently,up_set_current_regs/up_current_regs is only used for identifying interrupt contexts that do not match their names.